### DISCUSSION TITLE
SSL certificate invalid for non-vercel domain but valid for vercel

### DISCUSSION THREAD

**Discussion by nachiket87** [vercel/community#3157] [Help]
### Summary

I have a nextjs site deployed to two separate domains.

a .com domain from google domains and a .ca domain purchased from Vercel.

My workplace computer does not permit the .com domain saying the SSL cert is invalid but the .ca domain works correctly.

Is there a way to renew the SSL certificate or fixing the invalid one for the .com domain?

### Example

www.stopricksnow.com

### Steps to Reproduce

visit www.stopricksnow.com

then visit www.stopricks.ca

compare the SSL certs.

**Comment by amyegan** [comment]
Hi @nachiket87. Are you still having this problem? I'm not getting a warning about SSL cert at the `.com` address. I see Let's Encrypt certificates for both.

**Reply by nachiket87** [reply]
> Hi @nachiket87. Are you still having this problem? I'm not getting a warning about SSL cert at the `.com` address. I see Let's Encrypt certificates for both.

Hi @amyegan! Thanks for responding. I'm getting the invalid SSL on my work computer that has enchanted security. However, this doesn't occur on the .ca domain.

**Reply by nachiket87** [reply]
It seems like it's reversed now. 

www.stopricks.ca is getting blocked and www.stopricksnow.com is now valid.

**Reply by smaeda-ks** [reply]
as far as I can tell, your cert has been issued correctly and has no issues:

```
$ sslscan www.stopricks.ca
Version: 2.0.16
OpenSSL 3.1.1 30 May 2023

Connected to 76.76.21.164

Testing SSL server www.stopricks.ca on port 443 using SNI name www.stopricks.ca

  SSL/TLS Protocols:
SSLv2     disabled
SSLv3     disabled
TLSv1.0   disabled
TLSv1.1   disabled
TLSv1.2   enabled
TLSv1.3   enabled

  TLS Fallback SCSV:
Server supports TLS Fallback SCSV

  TLS renegotiation:
Session renegotiation not supported

  TLS Compression:
OpenSSL version does not support compression
Rebuild with zlib1g-dev package for zlib support

  Heartbleed:
TLSv1.3 not vulnerable to heartbleed
TLSv1.2 not vulnerable to heartbleed

  Supported Server Cipher(s):
Preferred TLSv1.3  256 bits  TLS_CHACHA20_POLY1305_SHA256  Curve 25519 DHE 253
Accepted  TLSv1.3  128 bits  TLS_AES_128_GCM_SHA256        Curve 25519 DHE 253
Accepted  TLSv1.3  256 bits  TLS_AES_256_GCM_SHA384        Curve 25519 DHE 253
Preferred TLSv1.2  256 bits  ECDHE-RSA-CHACHA20-POLY1305   Curve 25519 DHE 253
Accepted  TLSv1.2  128 bits  ECDHE-RSA-AES128-GCM-SHA256   Curve 25519 DHE 253
Accepted  TLSv1.2  256 bits  ECDHE-RSA-AES256-GCM-SHA384   Curve 25519 DHE 253
Accepted  TLSv1.2  256 bits  DHE-RSA-AES256-GCM-SHA384     DHE 2048 bits

  Server Key Exchange Group(s):
TLSv1.3  128 bits  secp256r1 (NIST P-256)
TLSv1.3  192 bits  secp384r1 (NIST P-384)
TLSv1.3  260 bits  secp521r1 (NIST P-521)
TLSv1.3  128 bits  x25519
TLSv1.2  128 bits  secp256r1 (NIST P-256)
TLSv1.2  192 bits  secp384r1 (NIST P-384)
TLSv1.2  260 bits  secp521r1 (NIST P-521)
TLSv1.2  128 bits  x25519

  SSL Certificate:
Signature Algorithm: sha256WithRSAEncryption
RSA Key Strength:    2048

Subject:  *.stopricks.ca
Altnames: DNS:*.stopricks.ca, DNS:stopricks.ca
Issuer:   R3

Not valid before: Jul  6 19:44:35 2023 GMT
Not valid after:  Oct  4 19:44:34 2023 GMT
```
